src/renderer/app-bridge.ts

```diff
@@ -153,7 +154,13 @@ export class AppBridge {
                     ssInstance.handleMessageEvents(data);
                 }
                 break;
-        }
+            case apiCmds.registerLogRetriever:
+                ssf.registerLogRetriever(this.callbackHandlers.onRegisterLogRetrieverCallback);
+                break;
+            case apiCmds.logReceiver:
+                ssf.logReceiver( data.logs );
```

Comment:
Extra blank space is not required for arguments

Reply:
done